src/main/java/jp/ikedam/jenkins/plugins/extensible_choice_parameter/SystemGroovyChoiceListProvider.java

```diff
@@ -117,11 +118,17 @@ public String getDisplayName()
          * @param scriptText
          * @return the selection of a default choice
          */
-        public ListBoxModel doFillDefaultChoiceItems(StaplerRequest req, @QueryParameter String scriptText, @QueryParameter boolean usePredefinedVariables)
+        public ListBoxModel doFillDefaultChoiceItems(StaplerRequest req, @QueryParameter String script, @QueryParameter boolean sandbox, @QueryParameter boolean usePredefinedVariables)

```

Comment:
`String script` and `boolean sandbox` are fields of script-security.
Though I know it's a really bad idea to depend on definitions of script-security, `SystemGroovyChoiceListProvider` would get difficult to use without this feature.